doc/gb-model/methodology.rst

```diff
@@ -0,0 +1,315 @@
+3. distributing the total constraint cost using the relative congestion rent delta of each boundary.
+
+This distribution can be applied hourly or weekly.
+The methodology prefers weekly as hourly includes fluctuations that are within the (re-)dispatch rolling horizon window, in which generators can bid/offer with perfect foresight on congestion for the whole day.
+
+Investment cost
+---------------
+
+To calculate the NPV of a system with a reinforcement option, the constraint costs is combined with the option's capital cost, assuming a 40-year project lifetime.
+To compare it with an annual constraint cost, it is amortised over its life by applying the `Spackman approach <https://www.ofgem.gov.uk/sites/default/files/docs/2011/10/discounting-for-cost-benefit-analysis-involving-private-investment-but-public-benefit.pdf>`_, using the TO's WACC to annualise and HM Treasury's Social Time Preferential Rate to discount all costs and benefits.
+It is an approach that is stated as suitable for projects with private financing but public benefits.
+
+Delay costs
+-----------
+
+It is possible for options to be delayed from their Earliest In Service Date (EISD).
+This usually comes at a cost, defined by the TO, which is added into the option total cost.
+
+Societal Cost of Carbon
+-----------------------
+
+The societal cost of carbon comes into play if operating carbon emissions reduce as a result of a reinforcement option.
+In such cases, an economic benefit equal to the quantity of reduced emissions (compared to the base case run?) is added to the NPV calculation.
+`Traded carbon values <https://www.gov.uk/government/collections/carbon-valuation--2>`_ are likely used for this calculation.
+
+.. note::
+
+  This does not appear in the NOA 2021-22 methodology, only the tCSNP, so we probably do not need to account for it to reproduce the NOA 2021-22.
+
+Modelling steps
+===============
+
+.. raw:: html
+
+   <iframe src="../_static/existing_methodology_modelling_steps.drawio.html" style="width: 100%; height: 300px; border: none;"></iframe>
+
+   Graphical representation of our understanding of the NOA Refresh 2021-22 modelling steps.
+
+
+The modelling process involves adding candidates one a time in a pre-defined priority order for each specific GB region.
+As :ref:`aforementioned <method-existing-boundary-capabilities>`, reinforcements in other GB regions are fixed based on the most recent NOA report (here: 2020-2021).
+Regions are arbitrarily defined.
+The only criterion is that no option may appear in more than one region.
+
+The method for a region goes as follows:
+
+1. Re-dispatch run is undertaken with all proposed options disabled, to generate a *base case* which defines the reinforcement requirements on each boundary in the region for the first year (2023).
+2. All reinforcement options are ordered by priority to satisfy the requirements (by EISD? by magnitude of requirement met?).
```

Comment:
should we leave the questions in?

Reply:
Yeah, I'm not sure where to store open questions...